dev/src/dev.clj

```diff
@@ -22,3 +37,19 @@
   (doseq [ns-name ns-names]
     (require ns-name :reload))
   (expectations/run-tests ns-names))
+
+(defmacro require-model
+  "Rather than requiring all models inn the ns declaration, make it easy to require the ones you need for your current session"
+  [model-sym]
+  `(require [(symbol (str "metabase.models." (quote ~model-sym))) :as (quote ~model-sym)]))
+
+(defmacro with-permissions
+  [permissions & body]
+  `(binding [api-common/*current-user-permissions-set* (delay ~permissions)]
+     ~@body))
+
+;; The linter will punch you in the face if you require a namespace without using anything from it, so here we pull a
+;; fast one on it. We want to require the namespaces so that they're within easy reach during REPL dev, without having
+;; to specify beforehand what exactly we want to use.
+(def appease-the-linter
+  [u/get-id mi/can-read?])
```

Comment:
the linters only complain if it's in the `ns` `:require` form so if you need to require it for some other reason just do a separate top-level require e.g.

```clj
(require '[metabase.models.interface :as mi])`
``.

Or just remove `dev` from the `:source-paths` for those linters since linting `dev` isn't something we really need to do